src/Action/LtiInstance/LtiInstanceDeleteAction.php

```diff
@@ -50,7 +50,7 @@ public function __invoke(string $ltiInstanceId): Response
         /** @var LtiInstance $ltiInstance */
         $ltiInstance = $this->repository->find($ltiInstanceId);
 
-        if (!$ltiInstance) {
+        if ($ltiInstance == null) {
```

Comment:
Please use strict comparison operator to check if value is null
https://phpbestpractices.org/#checking-for-null

Reply:
fixed